### ISSUE TITLE
interface becomes unresponsive while search is running

### ISSUE THREAD

**Issue by mmckegg** [ssbc/patchwork#441]
This is related to `hooks` (caching entire database). But also seems to be something else going on. 

At the very least we should suspend search when the tab is not active so it doesn't slow down other pages.

**Comment by mmckegg** [comment]
Gosh this is bad! Really need to figure out a better solution to this. I'm surprised more people aren't complaining 😟 

Should investigate full text search again. Maybe don't fall back to old search, and don't start indexing for fts until all the other indexes have completed. This might take the load off.

Or find a way to make the indexes work in the background (low priority) instead of soaking up all the cpu cycles.
